Automattic/jetpack
# Debug Helper IDC Simulator: display the last five remote requests

#### Changes proposed in this Pull Request:
* When testing IDCs, it can be useful to look at the url to verify that the IDC query args are added when expectd. It's also useful to see if WPCOM responded with the expected `idc_detected` and `migrated_for_idc` values. Display the last 5 request urls and the IDC response on the
IDC Simulator page.

#### Jetpack product discussion
* n/a

#### Does this pull request change what data or activity we track or use?
* no

#### Testing instructions:
1. Install, activate, and connect Jetpack.
2. Install and activate this branch of the Debug Helper plugin.
3. Enable the IDC Simulator module.
4. Navigate to the IDC Simulator page. When you reload the page, you should see a JITM request in the request table.
5. Continue reloading the page or doing other things on the site, and verify that a maximum of five remote requests are displayed.
